docs/BROWSERSETUP.md

```diff
@@ -0,0 +1,39 @@
+# Browser Setup for Testing
+It is possible to activate browser specific settings before test execution.  
+E.g. when a special proxy setting is needed for Firefox on a System Under Test (SUT) environment under Linux which is different to the proxy setting under Windows, it is possible to create a JSON-File called **"browserSetup.json** in the directory **src/test/resources** of the actual test project. These specified settings will be used for test execution.
+
+## Example structure of the JSON-File   
+![BrowserSetup](/images/browserSetup.png)
+
+### Browser Setup Elements
+The **browserSetup.json** file consists of several different Browser Setup Elements marked blue in the screenshot.  
+Each Element consists of    
+* an arbitrary chosen element name (e.g. in screenshot "firefox-section-without-os") 
+* an optional Browser Setting entry for the platform (os - marked yellow in the screenshot). 
+* an optional Browser Setting entry for the browser to test (browser - marked yellow in the screenshot). 
+* an optional Browser Setting entry for options (options - marked green in the screenshot).      
+* an optional Browser Setting entry for capabilities (capabilities - marked red in the screenshot). 
+
+
+### Browser Settings
+#### Key "os"  
+When a test should run with OS specific settings the key for "os" must be set, for the specific values please refer to the os specific values on the Selenium web site under [DesiredCapabilities].(https://github.com/SeleniumHQ/selenium/wiki/DesiredCapabilities)  
```

Comment:
When a test should run with OS-specific settings, the key for "os" must be set. For the specific values, please refer to the OS-specific values on the Selenium web site under [DesiredCapabilities](https://github.com/SeleniumHQ/selenium/wiki/DesiredCapabilities).